ext/gd/libgd/gdft.c

```diff
@@ -400,7 +400,7 @@ static void *fontFetch (char **error, void *key)
 	fontlist = gdEstrdup(a->fontlist);
 
 	/*
-	 * Must use gd_strtok_r becasuse strtok() isn't thread safe
+	 * Must use gd_strtok_r because strtok() isn't thread safe
```

Comment:
Same here.

Reply:
This one is [no longer present](https://github.com/libgd/libgd/blob/2be005f311232bc3d8a544f73ce8049d2b2fb885/src/gdft.c#L1807).